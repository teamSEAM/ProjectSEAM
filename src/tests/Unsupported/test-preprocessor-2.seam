#WHAT IS A COLON#
entity World:
	func start():
		screen.out("comment: test.")
		screen.init(100,100)
		screen.out("#DOES THIS PRINT?# MAYBE")
