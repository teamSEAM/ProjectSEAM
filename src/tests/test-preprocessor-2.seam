#WHAT IS A COLON#
entity World:
	function start():
		screen.out("comment: test.")
		screen.out("#DOES THIS PRINT?# MAYBE")
