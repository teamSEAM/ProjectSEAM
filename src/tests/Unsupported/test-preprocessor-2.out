comment: test.
#DOES THIS PRINT?# MAYBE
